libs/components/text-editor/src/lib/modules/text-editor/services/text-editor.service.ts

```diff
@@ -24,6 +24,10 @@ export class SkyTextEditorService {
     return this.#_editor;
   }
 
+  public get isInitialized(): boolean {
+    return this.#_editor !== undefined;
```

Comment:
Since this member is now being used outside its corresponding getter/setter, rename it to `#editor`.